src/elements/content-sidebar/Sidebar.js

```diff
@@ -287,12 +287,13 @@ class Sidebar extends React.Component<Props, State> {
         const onVersionHistoryClick = hasVersions ? this.handleVersionHistoryClick : this.props.onVersionHistoryClick;
         const styleClassName = classNames('be bcs', className, {
             'bcs-is-open': isOpen,
+            'bcs-wider': hasBoxAI,
```

Comment:
Consider using `bcs-is-wider` name to align with the `bcs-is-open` name that is already used.